xapian-letor/api/featuremanager_internal.cc

```diff
@@ -130,7 +128,6 @@ FeatureManager::Internal::load_relevance(const std::string & qrel_file)
         str = strtok(NULL, " ,.-");
         }
 
-        qrel1.insert(make_pair(token[0], Map1()));
```

Comment:
Did you mean to remove this rather than adjust it to insert a new `map<string, int>`?